### PR TITLE
[CMLG-015] Table Pagination Styling

### PR DESCRIPTION
**Pull request by annithinggoes**
Issue:
Pagination has no styling and no dark mode

Solution:
Added styling and fixed problems that appeared with table height and pagination width using flexboxes.

Risk:
/

Reviewed by:
Annie, Eileen, Dave, Yujia, Kevin

### CODE REVIEW

File: src/components/css/Pagination.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+   background-color: white;
+}
+
+.page-item.active .page-link {
+   color: black;
+   background-color: white;
+   text-decoration: underline;
+}
+
+.page-link:hover {
```
Comment (zyan225) [inline src/components/css/Pagination.css:20]: currently, if the user hovers the page that is currently displayed, this hover effect doesn't seem to work. 
e.g. only works on page 8 not page 7, is this the expected behaviour? 
图片
图片

File: src/components/css/Pagination.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.page-link {
+   margin: 1px; 
+   border: none;
+   color: gray;
+   background-color: white;
+}
+
+.page-item.active .page-link {
```
Comment (zyan225) [inline src/components/css/Pagination.css:14]: one thing I notice is that (may not actually related to this line), the page that gets a blue border may not be the page underlined. Steps: click page 5 -> click page 7, can see the page 9 gets a blue border. 
图片
图片

**Review by zyan225 — COMMENTED** [review]
Nice!!!!! well done.
Just some little stuff that needs to be changed. Have a look at my comments.

File: src/components/css/SearchPage.css [hunk 3]
```diff
@@ -1,12 +1,20 @@
...
+    overflow-y: auto;
 }
 
 .table-div {
     overflow: scroll;
-    height: 100%;
+    flex: 1;
+}
+
+.pagination {
```
[the remark this answers is not in the document]
Comment (annithinggoes) [reply src/components/css/SearchPage.css:13]: this is for the positioning of the div on the SearchPage, the actual css in the Pagination.css is for the styling of the pagination itself

File: src/components/css/Pagination.css (hunk 1 above)
Comment (annithinggoes) [reply src/components/css/Pagination.css:20]: this is same behaviour as light mode

File: src/components/css/Pagination.css (hunk 2 above)
Comment (annithinggoes) [reply src/components/css/Pagination.css:14]: yea.....idk why it does that....ill have a look at it

File: src/components/css/Pagination.css (hunk 1 above)
Comment (zyan225) [reply src/components/css/Pagination.css:20]: but does it make more sense to allow the hover working on the active page? same for light mode

Comment (annithinggoes) [reply src/components/css/Pagination.css:20]: dk how would you like me to change it?

Comment (zyan225) [reply src/components/css/Pagination.css:20]: having the same hovering effect on the currently active page as well?

**Review by hiin3d55 — APPROVED** [review]

File: src/components/css/Pagination.css (hunk 1 above)
Comment (annithinggoes) [reply src/components/css/Pagination.css]: image
image

Implemented this now

**Review by zyan225 — APPROVED** [review]
Nice work Annie!!!! 
Just remember to rebase master before merging in, the latest commit in master that solves the problem of "changing rows per page doesn't change pagination" in not currently in your branch.

**Pull request merged**